lib/chef/knife/ssh.rb

```diff
@@ -74,6 +74,13 @@ class Ssh < Knife
         # without a value
         :default => false
 
+      option :ssh_password_for_sudo,
+        :short => "-S",
+        :long => "--ssh-password-for-sudo",
+        :description => "EXPERIMENTAL. If ssh password given as argument to -P and a sudo COMMAND is " +
+                        "specified, this password will be passed into sudo using the -S flag",
```

Comment:
this looks kind of ugly when its printed out as help because its so long. Is there an easy way to indent a second line of an option description or should I write a shorter description? Any thoughts on what that description should be?